src/main/java/org/opentripplanner/gtfs/mapping/FareProductMapper.java

```diff
@@ -0,0 +1,69 @@
+package org.opentripplanner.gtfs.mapping;
+
+import java.time.Duration;
+import java.util.Currency;
+import org.opentripplanner.ext.fares.model.FareContainer;
+import org.opentripplanner.ext.fares.model.FareProduct;
+import org.opentripplanner.ext.fares.model.RiderCategory;
+import org.opentripplanner.routing.core.Money;
+
+public class FareProductMapper {
+
+  public static int NOT_SET = -999;
```

Comment:
Use methods in OBA instead to check if the values are set

Reply:
Could you give me a hint where this is already done? I can't seem to find it.

Reply:
It seems OBA doesn't have the methods for the new Fare classes. See eg. https://github.com/OneBusAway/onebusaway-gtfs-modules/blob/0dba859a9319d61f18bff2dc696190b3433a36fa/onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/FareAttribute.java#L118

Reply:
Here is the OBA PR: https://github.com/OneBusAway/onebusaway-gtfs-modules/pull/203

I will use these methods when there is a release.